packages/mock/src/addMocksToSchema.ts

```diff
@@ -146,7 +146,7 @@ export function addMocksToSchema({
 
     if (defaultResolvedValue === undefined) {
       // any is used here because generateFieldValue is a private method at time of writing
-      return (mockStore as any).generateFieldValue(info.parentType.name, info.fieldName);
+      return (store as any).generateFieldValue(info.parentType.name, info.fieldName);
```

Comment:
Could you remove `mockStore` variable and move `createMockStore` call near `store ||`?

Reply:
👍  done

Reply:
Thank you 🙏 One last thing. Could you fix prettier issues? Then i'll merge and release.

Reply:
My bad, IDE was misconfigured - it's fixed! 🙂